### PR TITLE
Feature/Level-Menu

### PR DESCRIPTION
**Pull request by timjacksonm**
Level Menu.
Includes button navigation. Back button & Home button. Might not need both - just unsure when and how this component will be accessed.
User can select a difficulty and the container will display levels 1-10 with anchor links to go to that page.
Still needs localstorage setup to display correct badge data. Currently have Badge1, Badge2, Badge3 included as default for first 3 levels as a template to showcase looks. Let me know what you think.

### CODE REVIEW

File: src/components/LevelSelect/Levels.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      6: 'Incomplete',
+      7: 'Incomplete',
+      8: 'Incomplete',
+      9: 'Incomplete',
+      10: 'Incomplete',
+    },
+  });
+  const [levels, setLevels] = useState(Object.entries(data[props.selection]));
+  const renderStatus = (status) => {
+    if (status === 'Incomplete') {
```
Comment (rankoliang) [inline src/components/LevelSelect/Levels.js]: Better to use a switch/case statement here.

File: src/components/LevelSelect/Levels.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      return <Incomplete />;
+    }
+    if (status === 'Badge1') {
```
Comment (rankoliang) [inline src/components/LevelSelect/Levels.js]: Could be slightly more descriptive here, like `BlueComplete`, `RedComplete`, `Complete`

**Review by rankoliang — APPROVED** [review]
I have some minor things that you can change, but it looks good to me! Great work, I really like it. I think we could make it a modal later and have it pop up on the home screen, but it's not critical.

**Comment by timjacksonm** [comment]
Did a few more updates. Ready to be merged with new approval.

**Review by rankoliang — APPROVED** [review]
Looks good to me! 🚀

**Pull request merged**